docs/guide/exe_verilator.rst

```diff
@@ -1329,6 +1329,11 @@ Summary:
    :vlopt:`-Wwarn-style`.  Excludes some specialty warnings,
    i.e. IMPERFECTSCH.
 
+.. option:: -Werror-UNUSED
+
+   Promote all unused warning messages into error messages. This is equivalent to
+   ``-Werror-UNUSEDGENVAR -Werror-UNUSEDPARAM -Werror-UNUSEDSIGNAL``.
+
```

Comment:
Instead describe UNUSED under warnings section.

Reply:
I removed the warning there and added it to docs/guide/warnings.rst file